Bylaws.md

```diff
@@ -181,8 +189,8 @@ Interested Parties will not be required to pay anything for their participation
 
 The Forum shall maintain a Member Mail List and Member Web Site that are not accessible by the public. The following matters may be posted to the Member Mail List and Member Web Site:
 
- 1. Draft minutes of Forum meetings (both virtual and in-person, and including any sub-groups or committees) will be posted to the Member Mail List to allow Members and Associate Members to make sure they are being correctly reported.
-    Minutes will be considered final when approved at a subsequent Forum Meeting or Forum Teleconference; provided, however, that if there is no Forum Meeting or Forum Teleconference scheduled within three (3) weeks of the publication of the draft minutes. Final minutes will then be posted to the Public Mail List and Public Web Site. The Chair will, upon request, make redactions of any part of the public copy of the minutes identified as private or sensitive by either the information discloser or a member mentioned or affiliated with the subject of the information.
+ 1. Draft minutes of Forum meetings or Teleconferences (both virtual and in-person, and including any Forum-level sub-groups or committees) will be posted to the Member Mail List or a Member Mail list of a sub-group or committee, to allow Members and Associate Members to make sure they are being correctly reported.
```

Comment:
We might want to reconsider the addition of `Teleconferences` as here we state that a teleconference can be in-person, which contradicts the term (a conference with participants in different locations linked by telecommunication devices). 

Maybe we should simply talk about a `virtual meeting` and a `physical meeting` troughout the document? Or define a `meeting` as a `virtual` or `physical meeting` in the definitions.

Reply:
There are specific definitions for a Teleconference and a Forum Meeting. I don't feel we should make updates in those definitions. "Forum Meetings" can be "virtual" or "in-person" or even "hybrid". However, "Teleconferences" can only be "virtual". @vanbroup can you please suggest some better wording without the need to update the existing definitions?

Reply:
If we have Meetings (which will be physical) and Teleconferences which will be (virtual) we can simply the language as following: 
```suggestion
 1. Draft minutes of Meetings and Teleconferences (including any Forum-level sub-groups or committees) will be posted to the relevant Member Mail list, to allow Members and Associate Members to make sure they are being correctly reported.
```

Reply:
Here is the update taking parts of your proposed language. https://github.com/cabforum/forum/pull/32/commits/040c0e293382925323d0240cb0312d2308f00f4b.

Let me know if it works.

Reply:
This could work while it's still confusing and remains hard to read:

It first starts with **Forum Meetings** and follows with a statement that it's **not just forum meetings**, why not just state **meetings** instead if we don't want to restrict it anyway?

I also keep struggling over "**Member Mail List or a Member Mail list of a sub-group or committee**", simply stating "**the corresponding Member Mail list**" which should be sufficient to link it to the forum, sub-group, or committee.

Reply:
"Forum Meetings" is a capitalized term which is defined as "Face-to-face plenary meetings of Voting Members, Probational Certificate Issuers and Associate Members as scheduled from time to time, and does not include meetings such as Subcommittee, subgroup, committee, or PAG meetings."

I took another take. Please take a look at https://github.com/cabforum/forum/pull/32/commits/0796f0c777d99cec5be8dc2bb3994d100e854c90 and tell me if it makes things clearer.

Reply:
That reads a lot better! You might want to replace the comma with 'and' in the first part of the sentence.

> Draft minutes of Forum Meetings **and** Forum Teleconferences will be posted to the corresponding Member Mail List

Reply:
Done. Thanks for the feedback :-)